config/usage.cfg

```diff
@@ -772,24 +772,6 @@ setdesc "sortlist" "returns a sorted list (quicksort) based on the result of bod
 setdesc "stripcolors" "returns the string without any colors;^nexample: stripcolors ^"^^frhello ^^f[6252287]^"" "string"
 setdesc "addlocalop" "elevates the privilege of a user to a specified level locally;^nthis allows server administrators to ^"upgrade^" the access of people as they identify, on that particular server,^nuser_handle is the authkey handle of the user,^nflag is the privilege level {a = administrator m = moderator o = operator s = supporter}" "user_handle flag"
 setdesc "resetlocalop" "removes all temporary elevated local privileges;^nthis allows server administrators to reset the access given by addlocalop for all users, on that particular server"
-setdesc "ircaddchan" "joins a channel on a given IRC instance with automatic rejoin;^nname - name of the IRC instance,^nchannel - channel to join,^n[friendly] - optional friendly name of channel,^n[passkey] - optional passkey required for mode +k channels,^n[relay] - optional relay level (verbosity of output)" "name channel [friendly] [passkey] [relay]"
```

Comment:
@MoonPadUSer perhaps we should transfer those docstrings into our docs? You experienced yourself that setting up IRC isn't overly easy for servers. Could be part of an article series on server hosting.

Reply:
yes, agree, let's put that on the TODO-List